namerd/examples/zk.yaml

```diff
@@ -3,6 +3,13 @@ storage:
   pathPrefix: /dtabs
   hosts:
   - localhost:2181
+  authInfo:
+    scheme: digest
+    auth: user_123:password_123
```

Comment:
i guess we want to support this use case, but i imagine folks may not be comfortable putting their password in the clear in our config files. a better option would be reading an env var, not sure the best way to accomplish.